dspy/predict/predict.py

```diff
@@ -17,10 +17,10 @@
 
 class Predict(Module, Parameter):
     def __init__(self, signature, callbacks=None, **config):
+        super().__init__(callbacks=callbacks)
```

Comment:
Btw typical custom DSPy programs don't call `super().__init__(callbacks=callbacks)`.

Will that be an issue?

Reply:
Nothing will crash, but they will lose some perks from `dspy.Module`. We can follow what PyTorch does - recommending calling `super().__init__()` when customizing modules: https://docs.pytorch.org/tutorials/beginner/examples_nn/polynomial_module.html#pytorch-custom-nn-modules